src/org/sipdroid/sipua/SipdroidEngine.java

```diff
@@ -21,22 +22,22 @@
 
 package org.sipdroid.sipua;
 
-import java.io.IOException;

```

Comment:
please put the import statements back in the same order so it is easier to review